sdf/1.5/schema/types.xsd

```diff
@@ -31,7 +31,8 @@
   </xsd:simpleType>
 
   <xsd:simpleType name="time">
-    <xsd:restriction base="xsd:double">
+    <xsd:restriction base="xsd:string">
+      <xsd:pattern value="\d+ \d+"/>
```

Comment:
Could you also update the others as well

```suggestion
      <xsd:pattern value="\s*\d+\s+\d+\s*"/>
```

Reply:
@jennuine Hm, if we are worried about whitespace, wouldn't something like this be a bit more readable?

```
<xsd:restriction base="xsd:string">
  <xsd:whiteSpace value="collapse"/>
  <xsd:pattern value="\d+ \d+"/>
</xsd:restriction>
```

Reply:
Sounds good to me :+1:

Reply:
@jennuine Done.